### ISSUE TITLE
Noise injection and bounding box

### ISSUE THREAD

**Issue by anirudh-g** [alankbi/detecto#46]
It would be good to have option to natively inject noise in the dataset. For now, is there any workaround for this?
Also, it would be nice to have native data augmentation that also transforms the bounding boxes accordingly

**Comment by alankbi** [comment]
Maybe torchvision [transforms](https://pytorch.org/docs/stable/torchvision/transforms.html) can help you with that? As for augmentation that also transforms the bounding boxes, `RandomHorizontalFlip` and `Resize` from the above link are currently the ones Detecto supports.

**Comment by anirudh-g** [comment]
When I include ToTensor transform, the show_labelled_image displays the image with a modified color in a brown'ish way as below. Without that, the image is displayed fine. Is this to be expected?
<img width="71" alt="Annotation 2020-08-17 120637" src="https://user-images.githubusercontent.com/22273340/90389827-24715980-e082-11ea-94f6-f322331b4422.png">

**Comment by anirudh-g** [comment]
Also, when I try to predict the labels for a new input image, it shows 6% confidence score. This is happening only after I started using the transform method. Previously, I set it as as None ( I assume the basic totensor and normilisation will still be applied) and it was predicting without any issues

**Comment by alankbi** [comment]
When applying custom transforms, do you normalize at the end (see [this](https://github.com/alankbi/detecto#advanced-usage) example)? That might help with the plotting and with the accuracy.
